.github/actions/block-main-pr/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Block Main Branch PRs
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited      
+
+jobs:
```

Comment:
Is this a workflow or an action? I don't think actions can have `jobs` or `on` clauses, since they can only run inside a workflow that should specify those